test/air-demo.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
```

Comment:
I don't think it really makes sense to merge this upstream.

I can try to investigate a little more what this is for and see if we can perhaps get rid of its (or rename it to something more generic).

Reply:
PR to remove this here: https://github.com/desmosinc/mathquill/pull/165